core/src/main/java/org/radargun/config/VmArgs.java

```diff
@@ -139,10 +158,32 @@ private static void ensureArg(Collection<String> args, String arg) {
       @Property(doc = "Min memory", converter = SizeConverter.class)
       private Long min;
 
+      @Property(doc = "Eden memory", converter = SizeConverter.class)
+      private Long eden;
```

Comment:
This is misleading; -Xmn sets the size of whole new generation, which consists of eden AND survivor spaces. The space is split between Eden and Survivor spaces according to -XX:SurvivorRatio
Source: https://docs.oracle.com/cd/E19900-01/819-4742/abeik/index.html